src/Schema/SchemaFactory.php

```diff
@@ -153,7 +153,7 @@ public function newSchema( $name, $data ) {
 	 *
 	 * @return SchemaFinder
 	 */
-	public function newSchemaFinder( Store $store = null ) {
+	public function newSchemaFinder( Store $store = null ): SchemaFinder {
```

Comment:
In these cases you can remove the @return doc

Reply:
Thanks, I thought they were a little repetitive.